editor/js/libs/ui.three.js

```diff
@@ -2,10 +2,12 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-UI.Texture = function ( mapping ) {
+UI.Texture = function ( parameters ) {
 
 	UI.Element.call( this );
 
+	parameters = parameters || {};
```

Comment:
`ui.js` doesn't use this pattern.

do you mind adding `setEncoding( encoding )` to `UI.Texture` instead?